lib/compilers/dotnet.ts

```diff
@@ -162,11 +130,10 @@ class DotNetCompiler extends BaseCompiler {
         return this.compilerOptions;
     }
 
-    async runCrossgen2(compiler, execOptions, crossgen2Path, publishPath, dllPath, options, outputPath) {
+    async runCrossgen2(compiler, execOptions, crossgen2Path, references, dllPath, options, outputPath) {
         const crossgen2Options = [
-            crossgen2Path, '-r', path.join(publishPath, '*'), dllPath, '-o', 'CompilerExplorer.r2r.dll',
+            crossgen2Path, '-r', path.join(references, '*.dll'), dllPath, '-o', 'CompilerExplorer.r2r.dll',
             '--codegenopt', 'NgenDisasm=*', '--codegenopt', 'JitDiffableDasm=1', '--parallelism', '1',
-            '--inputbubble', '--compilebubblegenerics',
```

Comment:
These two arguments improve the codegen quality in #3325 and make it closer to JIT, why remove them?

Reply:
I didn't remove them, I am just bad a conflict resolving